src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets

```diff
@@ -111,6 +112,7 @@ Copyright (c) .NET Foundation. All rights reserved.
                                 DisableTransitiveFrameworkReferenceDownloads="$(DisableTransitiveFrameworkReferenceDownloads)"
                                 KnownCrossgen2Packs="@(KnownCrossgen2Pack)"
                                 KnownILCompilerPacks="@(KnownILCompilerPack)"
+                                KnownILLinkPacks="@(KnownILLinkPack)"
```

Comment:
Where is `KnownILLinkPack` consumed?  Is it supposed to be converted to a `PackageReference`?  How are the .props and .targets inside it (which I assume used to be in the `Microsoft.NET.ILLink.Tasks` SDK) imported?